### ISSUE TITLE
ZDCRatingCell appearance not working

### ISSUE THREAD

**Issue by eikebartels** [zendesk/zendesk_sdk_chat_ios#66]
The following methods are not working 

```
[[ZDCRatingCell appearance] setTitleColor:[UIColor redColor]]; 
[[ZDCRatingCell appearance] setTitleFont:[UIFont fontWithName:AVENIR_FONT_BOOK size:12]];
```

**Comment by tecknut** [comment]
@eikebartels Thanks for pointing this out. We had spotted this internally and a fix is already in merged. We'll keep this open till we release a patch.

**Comment by oarrabi** [comment]
Hello @eikebartels,

We just released `ZDCChat` version 1.2.1.1 which has a fix to this issue.
Please check it out and let us know.

Thanks,
Omar...

**Comment by oarrabi** [comment]
I am going to close this for now, if you are still experiencing the issue, please feel free to reopen it
